pkg/service/node.go

```diff
@@ -164,13 +164,13 @@ func (s *service) UnmountVolume(ctx context.Context, volumeID, exportPath string
 
 	log.Infof("UnmountVolume calling Unmount %s", target)
 	err = sysUnmount(target, 0)
-	if err != nil && !strings.Contains(err.Error(), "No such file") {
-		log.Errorf("could not unmount the target path: %s %s %s", volumeID, target, err.Error())
+	if err != nil && !strings.Contains(err.Error(), "no such file") {
+		log.Errorf("Could not Umount the target path: %s %s %s", volumeID, target, err.Error())
 		return err
 	}
 
 	err = osRemove(target)
-	if err != nil && !strings.Contains(err.Error(), "No such file") {
+	if err != nil && !strings.Contains(err.Error(), "no such file") {
```

Comment:
I think the error message of "No such file" is returned from os.Remove call and changing it to lower case would not match the if condition?

Reply:
Adjusted to use proper error handling.